### PR TITLE
Feature/multilabel learning

### PR DESCRIPTION
**Pull request by AXLMRIN**

### CODE REVIEW

**Comment by AXLMRIN** [comment]
Right, 

Large PR but imma try to break it down: 

- implement a bert model training for multilabel schemes. The main difference are:
  - the "labels" go from `[0,1,2,0,1]` to `[[1,0,0],[0,1,0],[0,0,1],...`
  - the model takes an additional argument `problem_type = "multi_label_classification" if self.training_kind == "multilabel" else None`
  - To turn logits into probabilities we use the sigmoid instead of the softmax 
  -  to go from probs to labels we use a threshold instead of "max prob"
  -  i implemented a function called `find_best_threshold` which basically optimizes the threshold on the F1 score
  - The metrics function had to change (see below) 
  - I save new parameters: 
    - `training_kind`: "multiclass" or "multilabel"
    - `labels`: the labels the model was trained on
    - `use_dichotomization`: bool
    - `label_for_dichotomization`: string; ex "right"
  - these changes were carried out in the `train_bert.py`, `train_bert_multilabel.py`. `train_bert_multilabel.py` Should work out with both multiclass and multilabel, but `train_bert` only works for multiclass

- I changed the routes to train a bert model
- -
